### PR TITLE
Hugo v0.76.3

### PR DESCRIPTION
**Pull request by felicianotech**

### CODE REVIEW

File: src/resources/page/page_matcher.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+package page
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/pkg/errors"
+
+	"github.com/gohugoio/hugo/hugofs/glob"
```
Comment (chayev) [inline src/resources/page/page_matcher.go:22]: Update this

File: src/go.mod [hunk 2]
```diff
@@ -4,28 +4,27 @@ require (
...
+	github.com/evanw/esbuild v0.7.9
 	github.com/fortytw2/leaktest v1.3.0
-	github.com/frankban/quicktest v1.10.2
+	github.com/frankban/quicktest v1.11.0
 	github.com/fsnotify/fsnotify v1.4.9
-	github.com/getkin/kin-openapi v0.22.0
+	github.com/getkin/kin-openapi v0.22.1
 	github.com/ghodss/yaml v1.0.0
 	github.com/gobwas/glob v0.2.3
+	github.com/gohugoio/hugo v0.76.3
```
Comment (chayev) [inline src/go.mod:23]: Does this need to be here?

**Review by chayev — CHANGES_REQUESTED** [review]
A few updates required, everything else looks good.

File: src/resources/page/page_matcher.go (hunk 1 above)
Comment (felicianotech) [reply src/resources/page/page_matcher.go]: Good catch. Fixed. Also, made a PR to catch this for us in the future. https://github.com/gothamhq/gotham/pull/142

File: src/go.mod (hunk 2 above)
Comment (felicianotech) [reply src/go.mod]: No, but this is here because of the other thing you pointed out.

**Review by chayev — APPROVED** [review]
LGTM!

**Pull request merged**